dev-python/flask-login/flask-login-0.4.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+# pypy3 compat not possible due to missing compatibility of dev-python/semantic_version
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} pypy )
+
+inherit distutils-r1
+
+DESCRIPTION="Login session support for Flask"
+HOMEPAGE="https://pypi.org/project/Flask-Login/"
+# Should be replaced with PyPi URI for next release, if possible
+# See https://github.com/maxcountryman/flask-login/pull/393
+SRC_URI="https://github.com/maxcountryman/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+# pypi tarball is missing tests
+# see https://github.com/maxcountryman/flask-uploads/issues/2
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+RDEPEND=">=dev-python/flask-0.10[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
```

Comment:
RDEPEND is only required at build time when we run tests, it should be moved to `test?`